draft-brunstrom-taps-impl.md

```diff
@@ -386,6 +425,17 @@ Implementations may select the criteria by which a leaf node is considered to be
 
 For protocol stacks with multiple handshakes, the decision becomes more nuanced. If the protocol stack involves both TLS and TCP, an implementation MAY determine that a leaf node is connected after the TCP handshake is complete, or it MAY wait for the TLS handshake to complete as well. The benefit of declaring completion when the TCP handshake finishes, and thus stopping the race for other branches of the tree, is that there will be less burden on the network from other connection attempts. On the other hand, by waiting until the TLS handshake is complete, an implementation avoids the scenario in which a TCP handshake completes quickly, but TLS negotiation is either very slow or fails altogether in particular network conditions or to a particular endpoint.
 
+### Protocol Configuration
+
+Once a connection is established and a protocol stack has been chosen, an implementation SHOULD configure any Protocol Properties given by the application which apply to the chosen protocol stack.
+
+Additionally, an implementation MAY configure a transport protocol according to following Application Intents:
+
+* Traffic Category, Capacity Profile:
+The implementation may map information about the anticipated traffic pattern or priority to a DSCP code point value, e.g., see [I-D.ietf-tsvwg-rtcweb-qos]. Furthermore, if an application indicates a preference for low latency, the implementation may disable the Nagle algorithm on a TCP connection.
```

Comment:
But this is now already written for the "instantaneous capacity profile". Maybe remove "traffic category, capacity profile" here, or write TBD in case the intention is to be broader than what the "instantaneous capacity profile" offers?

Reply:
Indeed the Capacity Profile in the current API draft already reflects most of what we specify here, although we may get broader in the future as we figure out the relation between Capacity Profile, our Timeliness Intent, and our Category Intent, which I think is a post-London discussion.
Nevertheless, I think it makes sense to mention Protocol Configuration aspects from the API draft in the Implementation draft, too, also for including the Capacity Profile. Do you agree with this in general?

Reply:
In general yes, but my point is that this text repeats pretty much what I've already written in the implementation draft - if you search for Nagle or DSCP you'll see it

Reply:
Oh, now I see, that was an oversight on our part, sorry.
I removed the part for now, broadening the concept is TBD post-London.